src/rpp/jni_cpp.cpp

```diff
@@ -63,6 +85,15 @@ namespace rpp { namespace jni {
         }
     }
 
+    void ClearException() noexcept
+    {
+        auto* env = getEnv();
+        if (env->ExceptionCheck()) {
+            env->ExceptionDescribe();

```

Comment:
Should not describe it, because this will print out the exception into global log, which is not something we want